app/charts/shared/chart-helpers.tsx

```diff
@@ -212,12 +248,16 @@ export const extractChartConfigComponentIris = (chartConfig: ChartConfig) => {
     });
   }
 
-  return uniq(
-    [...fieldIris, ...additionalFieldIris, ...filterIris, ...IFIris].filter(
-      Boolean
+  return (
+    uniq(
+      [...fieldIris, ...additionalFieldIris, ...filterIris, ...IFIris].filter(
+        Boolean
+      )
     )
-    // Important so the order is consistent when querying.
-  ).sort();
+      .filter((d) => d !== "joinBy")
```

Comment:
Nit: Maybe, this should be a constant ?

Reply:
At least, when changing to multiple joinBy, you can remove the constant and see directly the affected areas.